slashing-protection/src/main/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtection.java

```diff
@@ -61,27 +165,37 @@ public boolean maySignBlock(
           final long id = validatorId.get();
           final Optional<SignedBlock> existingBlock =
               signedBlocksDao.findExistingBlock(h, id, blockSlot);
+
           // same slot and signing_root is allowed for broadcasting previously signed block
           // otherwise if slot and different signing_root then this is a double block proposal
-          final boolean maySign =
-              existingBlock.isEmpty() || existingBlock.get().getSigningRoot().equals(signingRoot);
-          if (maySign) {
+          if (existingBlock.isEmpty() || existingBlock.get().getSigningRoot().equals(signingRoot)) {
             final SignedBlock signedBlock = new SignedBlock(id, blockSlot, signingRoot);
             signedBlocksDao.insertBlockProposal(h, signedBlock);
+            return true;
+          } else {
+            LOG.error(
+                "Eth2 slashing protection for {} detected double signed block {}",
```

Comment:
Given this class IS the SlashingProtection (and thus appear in the logs) - is the "Eth2 Slashing protection" prefix required in the log text?

Reply:
no don't think it is. done